defaults/main.yml

```diff
@@ -1,11 +1,136 @@
 ---
 # defaults file for peertube
-peertube_tld: localhost
+peertube_tld: '{{ inventory_hostname }}'
+#peertube_alias: []
```

Comment:
Default value must be ["{{ inventory_hostname }}"]

Reply:
By default aliases are not defined, if you have one we do it in the vars of the container.